test-network-nano-bash/peer1.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+#look for binaries in local dev environment /build/bin directory and then in local samples /bin directory
+export PATH="${PWD}"/../../fabric/build/bin:"${PWD}"/../bin:"$PATH"
+export FABRIC_CFG_PATH="${PWD}"/../config
+
+export FABRIC_LOGGING_SPEC=debug:cauthdsl,policies,msp,grpc,peer.gossip.mcs,gossip,leveldbhelper=info
+export CORE_PEER_TLS_ENABLED=true
+export CORE_PEER_TLS_CERT_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.crt
+export CORE_PEER_TLS_KEY_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/server.key
+export CORE_PEER_TLS_ROOTCERT_FILE="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/tls/ca.crt
+export CORE_PEER_ID=peer0.org1.example.com
+export CORE_PEER_ADDRESS=127.0.0.1:7051
+export CORE_PEER_LISTENADDRESS=127.0.0.1:7051
+export CORE_PEER_CHAINCODEADDRESS=host.docker.internal:7052
+export CORE_PEER_CHAINCODELISTENADDRESS=127.0.0.1:7052
+#bootstrap peer is the other peer in the same org
+export CORE_PEER_GOSSIP_BOOTSTRAP=127.0.0.1:7053
+export CORE_PEER_GOSSIP_EXTERNALENDPOINT=127.0.0.1:7051
+export CORE_PEER_LOCALMSPID=Org1MSP
+export CORE_PEER_MSPCONFIGPATH="${PWD}"/crypto-config/peerOrganizations/org1.example.com/peers/peer0.org1.example.com/msp
+export CORE_OPERATIONS_LISTENADDRESS=127.0.0.1:9443
+export CORE_PEER_FILESYSTEMPATH="${PWD}"/data/peer0.org1.example.com
+
```

Comment:
Had to add something like this for each peer
`export CORE_LEDGER_SNAPSHOTS_ROOTDIR="${PWD}/snapshots/peer1.org1.example.com"`

Reply:
done.